utility/src/main/java/org/oppia/android/util/parser/html/ListItemLeadingMarginSpan.kt

```diff
@@ -66,14 +69,19 @@ sealed class ListItemLeadingMarginSpan : LeadingMarginSpan {
           paint.style = Paint.Style.STROKE
           paint.strokeWidth = 2f
         }
-        canvas.drawCircle(trueX.toFloat(), yPosition, bulletRadius.toFloat(), paint)
+        val correctX = if (isRtl) canvas.width - trueX - 1 - layout!!.width else trueX
```

Comment:
Why is the extra ``-layout!!.width`` needed here? I'd expect just the ``canvas.width - trueX - 1`` to be sufficient for RTL.

Reply:
I removed this extra layout!!.width. And I tried to change leading margin but the text doesn't move towards left.

Reply:
I think this was fixed in the latest changes per the need to change HtmlParser.